recipes/wasmtime/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "0.28.0":
```

Comment:
0.29.0 is out

Reply:
FYI.
0.29.0 was out 20 days ago, and the fix for the shared mingw got in 13 days ago.

Reply:
We can still bump regardless ;)

Reply:
It looks like they release at the beginning of each month.
If we can postpone a few weeks, we don't need to forbid shared mingw.

Reply:
Version was updated